src/components/layouts/php-collection-page-layout.tsx

```diff
@@ -0,0 +1,1189 @@
+  lesson,
+  completedLessonSlugs,
+  index,
+}: {
+  lesson: any
+  completedLessonSlugs: any
+  index: number
+}) => {
+  const isComplete = completedLessonSlugs.includes(lesson.slug)
+  return (
+    <li key={lesson.slug}>
+      <div className="flex py-2 font-semibold leading-tight h-20">
+        <div className="flex items-center mr-2 space-x-2">
+          <div
+            className={`${
+              isComplete
+                ? 'text-blue-600 dark:text-green-400'
+                : 'text-gray-500 dark:text-gray-400'
+            } pt-px font-xs scale-75 font-normal w-4`}
+          >
+            {isComplete ? (
+              <CheckIcon className="w-6 h-6 -translate-x-2" />
+            ) : (
+              index + 1
+            )}
+          </div>
+          {lesson.icon_url && (
+            <div className="flex items-center flex-shrink-0 w-8">
+              <Image src={lesson.icon_url} width={24} height={24} />
+            </div>
+          )}
+        </div>
+        {lesson.path && (
+          <div className="flex flex-col ">
+            <div>
+              <Link href={lesson.path}>
+                <a className="text-lg font-semibold hover:underline hover:text-blue-600 dark:text-gray-100">
+                  {lesson.title}
+                </a>
+              </Link>
+            </div>
+            <div className="text-xs text-gray-700 dark:text-gray-500">
+              {convertTimeWithTitles(lesson.duration, {
+                showSeconds: true,
+              })}
+            </div>
+          </div>
+        )}
+      </div>
+    </li>
+  )
+}
+
+const ModuleCollection = ({module, children}: {module: any; children: any}) => {
+  return (
+    <div key={module.id}>
+      <h2 className="text-xl font-bold mt-4 mb-2">{module.title}</h2>
+      {(() => {
+        switch (true) {
+          case module.contentList.length < 5:
+            return (
+              <div className={'grid grid-cols-2'}>
+                <ul className="grid gap-4">{children}</ul>
+                <p className="mb-6 prose text-gray-900 dark:prose-dark md:prose-lg md:dark:prose-lg-dark dark:text-gray-100 dark:prose-a:text-blue-300 dark:hover:prose-a:text-blue-200 prose-a:text-blue-500 hover:prose-a-:text-blue-600 ">
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                  do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                  Ut enim ad minim veniam, quis nostrud exercitation ullamco
+                  laboris nisi ut aliquip ex ea commodo consequat. Duis aute
+                  irure dolor in reprehenderit in voluptate velit esse cillum
+                  dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+                  cupidatat non proident, sunt in culpa qui officia deserunt
+                  mollit anim id est laborum.
+                </p>
+              </div>
+            )
+          case module.contentList.length > 5 && module.contentList.length <= 10:
```

Comment:
Here we are switching on the length of the contentList. Items will be displayed differently depending on the number of them in the array. 

In the future we may also need to use conditionals for different content types being present in the contentList